ROOTFS/etc/waggle/sanity/fatal/static_psu_control.test

```diff
@@ -0,0 +1,159 @@
+{
+        if [ -d $gpiopath/gpio$acok ]; then
+                echo $acok > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg1 ]; then
+                echo $pg1 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg2 ]; then
+                echo $pg2 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg3 ]; then
+                echo $pg3 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg4 ]; then
+                echo $pg4 > $gpiopath/unexport
+        fi
+}
+trap gpio_cleanup EXIT
+
+
+# Not my code, taken from Joe's PSU test
+
+# export the GPIOs and set to input
+if [ ! -d $gpiopath/gpio$acok ]; then
+        echo $acok > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$acok/direction
+if [ ! -d $gpiopath/gpio$pg1 ]; then
+        echo $pg1 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg1/direction
+if [ ! -d $gpiopath/gpio$pg2 ]; then
+        echo $pg2 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg2/direction
+if [ ! -d $gpiopath/gpio$pg3 ]; then
+        echo $pg3 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg3/direction
+if [ ! -d $gpiopath/gpio$pg4 ]; then
+        echo $pg4 > $gpiopath/export
+fi
+echo "in" > $gpiopath/gpio$pg4/direction
+
+
+# PG2 is the Edge processor, and should correspond to whether it is active in the manifest
+isEPPresent=$(cat /etc/waggle/node_manifest.json | jq .psu.ep_supply.present)
+
+# PG3/PG4 are bound, and should be treated as one AND'ed value. If they aren't AND'ed, big trouble. This AND'ed value should correspond to the peripherals in the manifest.
+isPHPresent=$(cat /etc/waggle/node_manifest.json | jq .psu.ph_supply.present )
+
+# check that we isEPPresent and isPHPresent have values
+
+if [[ $isPHPresent == "" ]];
+then 
+        echo "$pf: Could not get state of PH from manifest FAIL"
+fi
+
+if [[ $isEPPresent == "" ]];
+then 
+        echo "$pf: Could not get state of EP from manifest FAIL"
+fi
+
+
+
+echo "Node EP availability: $isEPPresent"
+echo "Node PH availability: $isPHPresent"
+
+
+bool_to_int() { 
+
+        if $1 -eq "true";
+        then echo "1"
+        else 
+        echo "0"
+        fi
+
+}
+
+
+# PG1 and ACOK Should always be high
+# Error codes 45 and 46 are respectively for failure of ACOK and PG1 
+# Error code 47 is for misaligned EP and PG2
+# Error code 48 is for misaligned PG3 and PG4
+# Error code 49 is for misaligned EP and PG3/PG4
+
+if ! gpio_high $acok; then
+        echo "$pf: ACOK3 Not powered FAIL"
+        exit 45
+fi
+
+if ! gpio_high $pg1; then
+        echo "$pf: PG1  Not powered FAIL"
+        exit 46
+fi
+
+
+# comparing EPSupply
+
+if [[  $(bool_to_int $isEPPresent) ==  $(gpio_high $pg2) ]];
+then 
+        echo "$pf: EP is $(bool_to_int $isEPPresent) in manifest, but PG2 is set to $? FAIL"
+        exit 47
+fi
+
+# make sure both pg3 and pg4 are the same
+if [[ $(gpio_high $pg3) != $(gpio_high $pg3) ]]; 
+then 
+        echo "$pf: PG3 and PG4 are not aligned FAIL"
```

Comment:
this true.. but what are the values of PG3 and PG4?  I know they are not the same, but what are they?